app/bundles/EmailBundle/EventListener/TokenSubscriber.php

```diff
@@ -73,23 +74,38 @@ public function onTokenReplacement(TokenReplacementEvent $event)
         }
 
         $lead      = $event->getLead();
+        $tokens    = $clickthrough['tokens'];
```

Comment:
I don't see any other place where EmailEvents::TOKEN_REPLACEMENT is triggered than at line 59 of this file. I don't see it in CampaignBundle